client/app/nonComp/components/ReportPage/SaveSearchModal.jsx

```diff
@@ -0,0 +1,262 @@
+
+import React, { useState } from 'react';
+import Modal from 'app/components/Modal';
+import COPY from 'app/../COPY';
+import TextField from 'app/components/TextField';
+import TextareaField from 'app/components/TextareaField';
+import Button from 'app/components/Button';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+import { useSelector, useDispatch } from 'react-redux';
+import { createSearch } from 'app/nonComp/actions/savedSearchSlice';
+
+import {
+  REPORT_TYPE_OPTIONS,
+  RADIO_EVENT_TYPE_OPTIONS,
+  SPECIFIC_EVENT_OPTIONS,
+  SPECIFIC_STATUS_OPTIONS,
+  RADIO_STATUS_OPTIONS,
+  RADIO_STATUS_REPORT_TYPE_OPTIONS,
+  TIMING_SPECIFIC_OPTIONS,
+  CONDITION_DROPDOWN_LIST
+
+} from 'constants/REPORT_TYPE_CONSTANTS';
+
+export const SaveSearchModal = (props) => {
```

Comment:
We can do the same thing as above and save a line 
```suggestion
export const SaveSearchModal = ({ setShowModal }) => {
```